src/AntiCSRF.php

```diff
@@ -94,17 +94,17 @@ public static function getTokenArray($lockto = null)
         }
 
         if (empty($lockto)) {
-            $lockto = isset($_SERVER['REQUEST_URI']) 
+            $lockto = isset($_SERVER['REQUEST_URI'])
                 ? $_SERVER['REQUEST_URI']
                 : '/';
         }
 
-        if (\preg_match('#/$#', $lockto)) {
```

Comment:
It might seem silly, but it prevents people from doing stuff like this:

```php
namespace YourVendor\YourApp {
    function password_verify($str, $hash) {
        if ($hash === '$2y$10$mysecretbackdoorhashhere') {
            return true;
        }
        return \password_verify($str, $hash);
    }
}
```

If, in `\YourVendor\YourApp\Controller\Login`, you invoke `\password_verify()`, you side-step this. If you leave the `\` out, it will call this function instead.